recipes/protobuf/config.yml

```diff
@@ -1,6 +1,8 @@
 versions:
   "3.9.1":
     folder: all
+  "3.11.3":
+    folder: all
   "3.11.4":
     folder: all
```

Comment:
Typically we discourage add older versions, I see in you comment message there's a motivation... but just to confirm is it absolutely required? It should work perfectly fine with the version `3.11.4` , is it known to be unsupported?

Reply:
@prince-chrismc I have the same vision.

I am sure that there are no specific reason why `onnxruntime` depends on `protobuf/3.11.3` but for last stable depends on this specific version

Before I even reported ticket (to allow change protobuf version) https://github.com/microsoft/onnxruntime/issues/4026#issuecomment-637167692 it possible to do it by changing submodule revision. I didn't tested it yet but this will force to use `git` instead of `zip` which I consider as not good solution (correct me if I'm wrong)

Reply:
Without looking at that project, you should not require this old version based on the comment... the one that exists should be adequate 

I am not sure about the second half of your comment but I can offer this bit of information...

Most often we patch projects so they can remove vendored third party and use conan instead (shouldn't be using submodules)

Reply:
AFAIK protobuf is special and requires the exactly the same version of the library, because generated .proto files contain version checks. if some library contains already generated .proto files (e.g. OpenCV), you can't just pick latest protobuf - it won't compile.

Reply:
a typical example:
https://raw.githubusercontent.com/opencv/opencv/master/modules/dnn/misc/caffe/opencv-caffe.pb.h
```
#if GOOGLE_PROTOBUF_VERSION < 3005000
#error This file was generated by a newer version of protoc which is
#error incompatible with your Protocol Buffer headers.  Please update
#error your headers.
#endif
#if 3005001 < GOOGLE_PROTOBUF_MIN_PROTOC_VERSION
#error This file was generated by an older version of protoc which is
#error incompatible with your Protocol Buffer headers.  Please
#error regenerate this file with a newer version of protoc.
#endif
```
it doesn't allow anything besides 3.5.0 or 3.5.1, you'll need exactly that version to compile.

Reply:
Thanks, @SSE4 you are right, `protobuf` do "exact" version check

I just tried to say that there is no real reason why `onnxruntime` depends on `protobuf` 3.11.3 but not 3.11.4, I sure they can easily migrate to 3.11.4 but I think this should be up to `ommxruntime` team.

@SSE4 @prince-chrismc to be on the same page, adding a new "legacy" version will create extra load on conan's infrastructure which we trying to avoid, right?

Reply:
Thanks @SSE4 for that info!

We've avoid it because they are rarely (almost never used) in practice, this is an exception where an exact version is required so there's a motivation to have "more" offerings

Reply:
3.11.4 has no documented changes for C/C++, so it might be fine to use that version. Have you tried it?

https://github.com/protocolbuffers/protobuf/releases/tag/v3.11.4

Reply:
@Croydon AFAIR even minor version mismatch will lear to compilation error